src/litdata/streaming/item_loader.py

```diff
@@ -569,8 +569,8 @@ def pre_load_chunk(self, chunk_index: int, chunk_filepath: str) -> None:
         """Logic to load the chunk in background to gain some time."""
         import polars as pl
 
-        if chunk_filepath not in self._df:
-            self._df[chunk_filepath] = pl.scan_parquet(chunk_filepath).collect()
+        if chunk_index not in self._df and os.path.exists(chunk_filepath):
```

Comment:
Let's maybe add an argument `pre_load_chunk=True/False` to the ParquertLoader to let the user optimize speed if they have enough RAM.